### ISSUE TITLE
TBVaccine in pipe mode is broken

### ISSUE THREAD

**Issue by frol** [skorokithakis/tbvaccine#10]
```bash
$ echo 1 | tbvaccine
Traceback (most recent call last):
  File "/mnt/storage/miniconda3/envs/CloudSML-Computational-Backend/bin/tbvaccine", line 11, in <module>
    sys.exit(main())
  File "/mnt/storage/miniconda3/envs/CloudSML-Computational-Backend/lib/python3.5/site-packages/tbvaccine/cli.py", line 29, in main
    output = tbv.process_line(line)
AttributeError: 'TBVaccine' object has no attribute 'process_line'
```

**Comment by skorokithakis** [comment]
Thank you, I recently refactored it and must have left a function. I'll fix it today.

**Comment by skorokithakis** [comment]
Released 0.1.3 with the fix.
